src/platform/posix/tcp_ip_channel.c

```diff
@@ -132,7 +144,7 @@ lf_ret_t TcpIpChannel_send(NetworkChannel *untyped_self, TaggedMessage *message)
   return LF_OK;
 }
 
-TaggedMessage *TcpIpChannel_receive(NetworkChannel *untyped_self) {
+static TaggedMessage *TcpIpChannel_receive(NetworkChannel *untyped_self) {
```

Comment:
I like static functions, but I like const pointers better, we should start using them also

Reply:
I agree ;)